README.md

```diff
@@ -66,16 +66,38 @@ We have the following two roles:
 
 ### Integration with GitHub
 
-To integrate with GitHub:
+#### GitHub App
+
+To integrate with GitHub using app authentication:
+
+1. **GitHub App**: Set up a GitHub app for Buildbot to enable GitHub user
+   authentication on the Buildbot dashboard.
+   Enable the following permissions:
+   - Contents: Read-only
+   - Metadata: Read-only
+   - Commit statuses: Read and write
```

Comment:
Ok. Found it.

Reply:
I have:
- commit statuses: RW
- contents: R
- issues: RW
- merge queues: RW
- metadata: R
- webhooks: RW

Reply:
I'm sure webhooks are needed for creation, the rest are kind of vague, ill check the endpoints